packages/nc-gui/components/smartsheet/toolbar/ColumnFilter.vue

```diff
@@ -100,6 +100,11 @@ watch(
   },
 )
 
+const getApplicableFilters = (id?: string) => {
+  const colType = (meta.value?.columnsById as Record<string, ColumnType>)?.[id ?? '']?.uidt
```

Comment:
in which case `id` would be not required?

Reply:
if user is not selected a column id will be undefined/null